hardhat_scripts/staging_redeem_queue_migration/tenderly/01_claim_all.ts

```
@@ -0,0 +1,154 @@
+
+    const balBefore = await retry(() => provider.getBalance(cur.recipient));
+    await (
+      await rmCaller["claimRedeemRequests(uint32[],uint32[])"]([id], [eventId], { gasLimit: 8_000_000 })
+    ).wait();
+    const paid = (await retry(() => provider.getBalance(cur.recipient))).sub(balBefore);
+    totalPaid = totalPaid.add(paid);
+    claimedCount += 1;
+
+    // Check against the pre-upgrade record rather than against what the chain says today.
+    const ref = pre.get(id);
+    let refRecipient = ref ? ref.recipient : "(created after the upgrade)";
```

ESLint

[error] 'refRecipient' is never reassigned. Use 'const' instead.

(prefer-const)